src/patterns/OcNavbar.vue

```diff
@@ -43,8 +43,8 @@ export default {
 ```jsx
   <oc-navbar tag="header" class="oc-topbar">
     <oc-navbar-item position="left">
-      <button class="oc-topbar-menu-burger" aria-label="Files">
-        <oc-icon name="menu" class="oc-topbar-menu-burger-icon" /><span
+      <button class="oc-topbar-menu-burger">
```

Comment:
from my pov this is wrongly design from a component perspective.
oc-button shall be used. oc button already has the option to have an icon inside and within such an oc-button component the aria labels shall be set properly.

correct my if I'm wrong ;-)

Reply:
aria-label is not necessary if the button has got an accessible label from another source. In this case it's `<button>this very text here</button>`. Therefore I removed the aria-label attribute. `<button aria-label="foo">foo</button>` would just be redundant.

I did not establish the direct use of `<button />`, if you're referring to this. Assumed it had to be this way here.